### ISSUE TITLE
Windows 10 Desktop ISO Issue

### ISSUE THREAD

**Issue by roflhouse** [cloudbase/windows-imaging-tools#379]
I tried using this app and hit a rather annoying problem. The Windows ISOs provided by Microsoft no longer work as expected by the tool. Namely, they no longer are shipped with install.wim files. As far as I can tell this is because Microsoft has started using a unified installer for its various versions, Home, Pro, etc. Now they are using install.esd files. This iso type is Provided via the "Media Creation Tool" which appears to be the only way to obtain iso files from Microsoft. 

Long story short, I don't believe there is a way to get Windows 10 ISOs in the format this tool is expecting anymore. From my very short googling, Install.wim files are only provided to specific version installers, aka a Pro specific installer.

There are some google hits about converting install.esd to install.wim files but that is going down a rabbit hole i don't wish to go down

**Comment by ader1990** [comment]
Hello, I need to get an ESD file to reproduce the issue, but the solution might be actually very simple.

You can try changing the ESD to a WIM:

```powershell
Dism /Export-image /SourceImageFile:C:\WimImage\install.esd /SourceIndex:1 `
    /DestinationImageFile:C:\WimImage\install.wim /Compress:max /CheckIntegrity

```

Thank you,
Adrian

**Comment by roflhouse** [comment]
You can get a copy of the new ISOs from Microsoft's tool at this website: https://www.microsoft.com/en-us/software-download/windows10.

I think there might be a way to use that command to modify the tool. It depends on what the wim file is used for.
